### DISCUSSION TITLE
Help with AsyncStream

### DISCUSSION THREAD

**Discussion by emil97-isaev** [tweepy/tweepy#1620] [Questions] [answered]
Hi! 
I'm trying to run AsyncStream Tweepy, but I ran into a problem

My code

```
from __future__ import absolute_import, print_function
from tweepy.streaming import Stream
from tweepy import OAuthHandler
from tweepy import Stream
from pprint import pprint
from tweepy.asynchronous import AsyncStream 
import asyncio

async def main():

    stream = StdOutListener(CONSUMER_KEY, CONSUMER_SECRET, ACCESS_TOKEN, ACCESS_TOKEN_SECRET)
    await stream.filter(follow=['1082189695252074496'])
    await asyncio.sleep(1.5)


class StdOutListener(AsyncStream):
    
    
    async def on_status(self, status):
        print(status.txt)

    async def on_error(self, status):
        print(status)
            
if __name__ == '__main__':
    asyncio.run(main())
```

When I run it in .py file, it doesn't work and returns the error "An HTTP: 420 error occurred in the stream". 
I also run the code in Jupiter, only instead of async io.run(main ()), I write await main(), it also returns this error, BUT the stream works and it returns a response. 

Why does it work in Jupiter, but does not work in the .py file. How can this be fixed?

**Comment by Harmon758** [comment, marked as answer]
A 420 HTTP status code indicates a Rate Limited error:
> The client has connected too frequently. For example, an endpoint returns this status if:
> - A client makes too many login attempts in a short period of time.
> - Too many copies of an application attempt to authenticate with the same credentials.

https://developer.twitter.com/en/docs/twitter-api/v1/tweets/filter-realtime/guides/connecting

**Reply by emil97-isaev** [reply]
Okay, thanks. Am I running an asynchronous stream correctly? For some reason, it only works in Jupiter Notebook?

**Reply by Harmon758** [reply]
`print(status.txt)` will raise an `AttributeError`, but besides that, the stream should connect fine.
I'm unable to reproduce any issues.
